cloudshell_utilities/.python-version

```diff
@@ -0,0 +1 @@
+3.7.3
```

Comment:
this doesn't seem related/necessary for this PR?

Reply:
Oops, got too keen with the commit. Removed.